rosdep/python.yaml

```diff
@@ -5636,6 +5636,9 @@ python3-docker:
   rhel:
     '7': ['python%{python3_pkgversion}-docker']
   ubuntu: [python3-docker]
+python3-docopt:
+  debian: [python3-docopt]
```

Comment:
Please add rules for [Fedora](https://fedora.pkgs.org/32/fedora-x86_64/python3-docopt-0.6.2-16.fc32.noarch.rpm.html), [Gentoo](https://packages.gentoo.org/packages/dev-python/docopt) and [Arch](https://www.archlinux.org/packages/community/any/python-docopt/).

Reply:
Thank you for the links!